src/main/kotlin/org/rust/lang/core/psi/ext/RsMembers.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.psi.ext
+
+import com.intellij.psi.StubBasedPsiElement
+import org.rust.lang.core.psi.*
+
+/** Returns all members including those produced by macros */
+val RsMembers.expandedMembers: List<RsAbstractable>
+    get() {
+        val members = mutableListOf<RsAbstractable>()
+
+        for (member in childrenSequence) {
+            when (member) {
+                is RsAbstractable -> members.add(member)
+                is RsMacroCall -> member.processExpansionRecursively {
```

Comment:
Maybe we could add addditional function to this place, so this `false` in the air will not look so strange?
Like this: 
```kotlin
private fun RsMacroCall.collectAbstractableMembersRecursively(): List<RsAbstractable> {
    val members = mutableListOf<RsAbstractable>()

    this.processExpansionRecursively { 
        if (it is RsAbstractable) {
            members.add(it)
        }

        false
    }

    return members
}
```
and then use it like 
```kotlin
... -> members.addAll(member.collectAbstractableMembersRecursively())
```